pygmt/datasets/earth_age.py

```diff
@@ -57,6 +59,21 @@ def load_earth_age(resolution="01d", region=None, registration=None):
     The :class:`xarray.DataArray` grid doesn't support slice operation, for
     Earth seafloor crustal age with resolutions of 5 arc-minutes or higher,
     which are stored as smaller tiles.
+
+    Examples
+    --------
+
+    >>> from pygmt.datasets import load_earth_age
+    >>> # load the default grid (gridline-registered 01d grid)
+    >>> grid = load_earth_age()
+    >>> # load the 30m grid with "gridline" registration
+    >>> grid = load_earth_age(resolution="30m", registration="gridline")
```

Comment:
I feel for the sake of clarity it's  good to state the unit in long form.

```suggestion
    >>> # load the 30 arc-minute grid with "gridline" registration
    >>> grid = load_earth_age(resolution="30m", registration="gridline")
```